src/Emulator/Cores/osx-properties.csproj

```diff
@@ -3,6 +3,7 @@
 <PropertyGroup>
   <CompilerPath>/usr/bin/clang</CompilerPath>
   <LinkerPath>/usr/bin/clang</LinkerPath>
+  <ArPath>ar</ArPath>
```

Comment:
For consistency, can you make it `/usr/bin/ar`?

Reply:
Changed.  I wasn't sure what it was called on osx, or where it was in the path.  I've now prefixed it with `/usr/bin/` as requested.

Reply:
I verified it's ok :+1: